cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/PackCommand.java

```diff
@@ -246,6 +246,18 @@ public void execute() {
         // Check package files are modified after last build
         boolean isPackageModified = isProjectUpdated(project);
 
+        // Checks if Package.md is present and issues a warning
+        Path packageMd = project.sourceRoot().resolve(ProjectConstants.PACKAGE_MD_FILE_NAME);
+        if (packageMd.toFile().exists()) {
+            String warning = """
```

Comment:
Let's use the https://github.com/ballerina-platform/ballerina-lang/blob/adec0634b84a1eb0b69d997c07207720b56a48c1/compiler/ballerina-tools-api/src/main/java/io/ballerina/tools/diagnostics/DiagnosticFactory.java#L41 to create a diagnostic

Reply:
Let's introduce DEPRECATED_DOC_FILE("BCE5601", "deprecated.doc.file") code for this

Reply:
So far, we haven't added any diagnostics in the Command classes. If it's an error we either throw a `ProjectException`. If it's a warning we just print them. 

I think printing the warning is fine. Otherwise we'll have to introduce diagnostics to `PackCommand`, move all the current error/warning printing  to diagnostics instead.

Reply:
Discussed this offline with @azinneera and decided to add the warning in the format of a diagnostic.

Reply:
@khadijahazward Let's remove the space before and after the warning message. It can be a simple string declaration.